### PR TITLE
DevWs for Most Target Limited containing containing ZLL RGBW Bulb

### PR DESCRIPTION
**Pull request by adurosmart**

### CODE REVIEW

**Comment by shrakuma** [comment]
@greens @MAblewiczS  can you please review this FP submission

File: devicetypes/smartthings/zll-rgbw-bulb.src/zll-rgbw-bulb.groovy [hunk 1]
```diff
@@ -64,9 +67,6 @@ metadata {
 		fingerprint profileId: "C05E", inClusters: "0000,0003,0004,0005,0006,0008,0300,1000", outClusters: "0019", manufacturer: "Philips", model: "LCT016", deviceJoinName: "Philips Hue A19"
 		fingerprint profileId: "C05E", inClusters: "0000,0003,0004,0005,0006,0008,0300,1000", outClusters: "0019", manufacturer: "Philips", model: "LST001", deviceJoinName: "Philips Hue Lightstrip"
 		fingerprint profileId: "C05E", inClusters: "0000,0003,0004,0005,0006,0008,0300,1000", outClusters: "0019", manufacturer: "Philips", model: "LST002", deviceJoinName: "Philips Hue Lightstrip"
-		
-		//XLSmart
-		fingerprint profileId: "C05E", manufacturer: "GLEDOPTO", model: "GL-B-001Z", deviceJoinName: "XLSmart E14 RGBW Light Bulb"
```
Comment (greens) [inline devicetypes/smartthings/zll-rgbw-bulb.src/zll-rgbw-bulb.groovy:69]: please don't delete existing fingerprints

File: devicetypes/smartthings/zll-rgbw-bulb.src/zll-rgbw-bulb.groovy [hunk 2]
```diff
@@ -30,12 +30,15 @@ metadata {
 		attribute "colorName", "string"
 
 		// Generic
-		fingerprint profileId: "C05E", deviceId: "0210", inClusters: "0006, 0008, 0300", deviceJoinName: "Generic RGBW Light"
```
Comment (greens) [inline devicetypes/smartthings/zll-rgbw-bulb.src/zll-rgbw-bulb.groovy:33]: please don't change existing fingerprints

**Comment by greens** [comment]
I think the issue (with this submission as well as others) is the branch is out of date with master. Please rebase.

**Comment by adurosmart** [comment]
No changes to the code, we just want to change the display name of the product, so can I close the request?
@greens

**Pull request closed without merging**